inbc-program/inbc/parser/source_app_parser.py

```diff
@@ -35,8 +35,8 @@ def application_add(args: argparse.Namespace) -> str:
 
     manifest += '<repo><repos>'
 
-    for source in args.sources:
-        manifest += '<source_pkg>' + source + '</source_pkg>'
+    source_tags = [f'<source_pkg>{source}</source_pkg>' for source in args.sources]
+    manifest += ''.join(source_tags)
```

Comment:
Use `str.join()` directly on the list comprehension for `source_tags` to avoid an unnecessary list creation. [medium]